test/game_teams_manager_test.rb

```diff
@@ -103,14 +103,14 @@ def test_worst_offense_dummy_file
     assert_equal "5", game_teams_manager.worst_offense
   end
 
-  def test_total_goals_by_team_dummy_file
```

Comment:
Looks like we need a test for create_goals_hash

Reply:
i wrote a test for that